src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -4,89 +4,83 @@
+		//defaultCommands: elevator, dt
+		//(pass in controller!)
 
-    setDefaultCommands();
-    setBindingsDriver();
-    setBindingsManipulator();
+    // setupAutos();
+		setDefaultCommands();
+		setBindingsDriver();
+		setBindingsManipulator();
   }
-
-  private void setDefaultCommands() {
+  
+	private void setDefaultCommands() {
     // drivetrain.setDefaultCommand(new TeleopDrive(
     //   drivetrain,
-    //   () -> ProcessedAxisValue(driverController, Axis.kLeftY)),
-    //   () -> ProcessedAxisValue(driverController, Axis.kLeftX)),
-    //   () -> ProcessedAxisValue(driverController, Axis.kRightX)),
-    //   () -> driverController.getRawButton(OI.Driver.slowDriveButton)
+    //   (DoubleSupplier) () -> ProcessedAxisValue(driverController, Axis.kLeftY),
+    //   (DoubleSupplier) () -> ProcessedAxisValue(driverController, Axis.kLeftX),
+    //   (DoubleSupplier) () -> ProcessedAxisValue(driverController, Axis.kRightX),
+    //   (BooleanSupplier)() -> driverController.getRawButton(OI.Driver.slowDriveButton)
     // ));
   }
   private void setBindingsDriver() {}
-  private void setBindingsManipulator() {}
+  
+  private void setBindingsManipulator() {
+  //have the trigger and button bindings here call the Intake, Shoot, and Eject commands
 
-  public Command getAutonomousCommand() {
-    return Commands.print("No autonomous command configured");
-  }
+  //NEW BINDINGS(easier for manipulator)
+  //Xbox right trigger axis -> Shoot
+  //Xbox left trigger axis -> Intake
+  //Xbox right bumper button -> Speaker (arm position)
+  //Xbox left bumper button -> Amp (arm position and RPM and Eject)
+  //Xbox X button -> Intake(arm position)
 
-  /**
-   * Flips an axis' Y coordinates upside down, but only if the select axis is a joystick axis
-   * 
-   * @param hid The controller/plane joystick the axis is on
-   * @param axis The processed axis
-   * @return The processed value.
-   */
-  private double getStickValue(GenericHID hid, Axis axis) {
-    return hid.getRawAxis(axis.value) * (axis == Axis.kLeftY || axis == Axis.kRightY ? -1 : 1);
-  }
-  /**
-   * Processes an input from the joystick into a value between -1 and 1, sinusoidally instead of linearly
-   * 
-   * @param value The value to be processed.
-   * @return The processed value.
-   */
-  private double inputProcessing(double value) {
-    double processedInput;
-    // processedInput =
-    // (((1-Math.cos(value*Math.PI))/2)*((1-Math.cos(value*Math.PI))/2))*(value/Math.abs(value));
-    processedInput = Math.copySign(((1 - Math.cos(value * Math.PI)) / 2) * ((1 - Math.cos(value * Math.PI)) / 2),
-        value);
-    return processedInput;
+    //IMPLEMENTING BINDINGS WHEN MERGED WITH ARM
+    /*/Eject also for AMP/*/
+    new JoystickButton(manipulatorController, OI.Manipulator.AMP_BUTTON).onTrue(new Eject(intakeShooter));
+
+    /*/Shooting/*/
+    new JoystickButton(manipulatorController, OI.Manipulator.SHOOTER_BUTTON.value).onTrue(new SequentialCommandGroup(
```

Comment:
why `.value`? Also use `import static org.carlmontrobotics.Constants.OI.*` to avoid having to type out `OI.Manipulator`